nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js

```diff
@@ -76,6 +79,7 @@ export function GatewayJsonConfig() {
             variant: 'success',
           });
           setError('');
+          history.goBack();
```

Comment:
minor comment: You can get rid of the status in the setState, mutable_lte_gateway doesn't need status attribute